src/MediatR/AsyncRequestHandler.cs

```diff
@@ -11,7 +11,7 @@ public abstract class AsyncRequestHandler<TMessage> : IAsyncRequestHandler<TMess
     {
         public async Task<Unit> Handle(TMessage message)
         {
-            await HandleCore(message);
+            await HandleCore(message).ConfigureAwait(false);
```

Comment:
@jbogard Should the users be able to access things which require context? i.e. user interface components, http context...